docs/howto-run-microservices-locally.md

```diff
@@ -1,10 +1,10 @@
-# Run the Azure Industrial IoT Services locally
+# Run the OPC Twin Microservices locally
 
-The micro services contained in this repository provide a REST like API on top of our Azure Industrial IoT components, for example our [OPC UA components](https://github.com/Azure/azure-iiot-opc-ua).  
+The OPC Twin Microservices contained in this repository provide a REST like API on top of our Azure Industrial IoT components, for example our [OPC UA components](https://github.com/Azure/azure-iiot-opc-ua).  
```

Comment:
Leave Azure Industrial IoT instead of making it OPC Twin.  As we discussed - we want the "services" and "modules" separation, but only use "OPC Twin when we talk about the OPC Twin Microservice.  

Are you going to break out the modules and services API folders in another commit?

Reply:
If not - its ok - we can do that later.

Reply:
Yes, I am doing that in next commit.